### PR TITLE
Fix typos in lib/{eunit,ftp}

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

File: lib/ftp/doc/archive/rfc959.txt [hunk 1]
```diff
@@ -78,7 +78,7 @@ File Transfer Protocol
       RFC 354 obsoleted RFCs 264 and 265.  The File Transfer Protocol
       was now defined as a protocol for file transfer between HOSTs on
       the ARPANET, with the primary function of FTP defined as
-      transfering files efficiently and reliably among hosts and
+      transferring files efficiently and reliably among hosts and
```
Comment (garazdawi) [inline lib/ftp/doc/archive/rfc959.txt:81]: The RFC is a copy of the upstream RFC, so leave that text as it is. (Yes, the upstream RFC has spelling mistakes)

Comment (kianmeng) [reply lib/ftp/doc/archive/rfc959.txt]: > The RFC is a copy of the upstream RFC, so leave that text as it is. 

@garazdawi Fixed.

> (Yes, the upstream RFC has spelling mistakes)

:rofl:

Comment (garazdawi) [reply lib/ftp/doc/archive/rfc959.txt]: You have done more changes in the RFC that needs to be reverted.

Comment (kianmeng) [reply lib/ftp/doc/archive/rfc959.txt]: My bad. Fixed.

File: lib/ftp/test/erl_make_certs.erl [hunk 2]
```diff
@@ -44,7 +44,7 @@
 %%      {title, Title}
 %%      {dnQualifer, DnQ}
 %%   issuer = {Issuer, IssuerKey}                   true (i.e. a ca cert is created) 
-%%                                                  (obs IssuerKey migth be {Key, Password}
+%%                                                  (obs IssuerKey might be {Key, Password}
```
Comment (HansN) [inline lib/ftp/test/erl_make_certs.erl:47]:
```suggestion
%%                                                  (Note! The IssuerKey might be {Key, Password}
```

File: lib/ftp/test/ftp_SUITE.erl [hunk 3]
```diff
@@ -948,7 +948,7 @@ unexpected_call(Config) when is_list(Config) ->
     Flag =  process_flag(trap_exit, true),
     Pid = proplists:get_value(ftp, Config),
 
-    %% Serious programming fault, connetion will be shut down
+    %% Serious programming fault, connection will be shut down
```
Comment (HansN) [inline lib/ftp/test/ftp_SUITE.erl:951]:
```suggestion
    %% A serious programming fault, the connection will be shut down
```

File: lib/ftp/test/ftp_SUITE.erl [hunk 4]
```diff
@@ -965,7 +965,7 @@ unexpected_cast()->
 unexpected_cast(Config) when is_list(Config) ->
     Flag = process_flag(trap_exit, true),
     Pid = proplists:get_value(ftp, Config),
-    %% Serious programming fault, connetion will be shut down
+    %% Serious programming fault, connection will be shut down
```
Comment (HansN) [inline lib/ftp/test/ftp_SUITE.erl:968]:
```suggestion
     %% A serious programming fault, the connection will be shut down
```

File: lib/ftp/test/ftp_format_SUITE.erl [hunk 5]
```diff
@@ -64,7 +64,7 @@ end_per_testcase(_, _) ->
 %%-------------------------------------------------------------------------
 
 ftp_150() ->
-    [{doc, "Especially check that respons can be devided in a random place."}].
+    [{doc, "Especially check that response can be divided in a random place."}].
```
Comment (HansN) [inline lib/ftp/test/ftp_format_SUITE.erl:67]:
```suggestion
    [{doc, "Especially check that a response can be divided in a random place."}].
```

File: lib/ftp/test/ftp_format_SUITE.erl [hunk 6]
```diff
@@ -75,7 +75,7 @@ ftp_150(Config) when is_list(Config) ->
     {pos_prel, _} = ftp_response:interpret(Msg).
    
 ftp_200() ->
-    [{doc, "Especially check that respons can be devided after the first status "
+    [{doc, "Especially check that response can be divided after the first status "
```
Comment (HansN) [inline lib/ftp/test/ftp_format_SUITE.erl:78]:
```suggestion
    [{doc, "Especially check that a response can be divided after the first status "
```

File: lib/ftp/test/ftp_format_SUITE.erl [hunk 7]
```diff
@@ -86,7 +86,7 @@ ftp_200(Config) when is_list(Config) ->
     ok.
 
 ftp_220() ->
-    [{doc, "Especially check that respons can be devided after the "
+    [{doc, "Especially check that response can be divided after the "
```
Comment (HansN) [inline lib/ftp/test/ftp_format_SUITE.erl:89]:
```suggestion
    [{doc, "Especially check that a response can be divided after the "
```

File: lib/ftp/test/ftp_format_SUITE.erl [hunk 8]
```diff
@@ -86,7 +86,7 @@ ftp_200(Config) when is_list(Config) ->
...
      "first with space "}].
```
Comment (HansN) [inline lib/ftp/test/ftp_format_SUITE.erl:90]: Should it be:
```suggestion
     "first white space "}].
```
?

File: lib/ftp/test/ftp_format_SUITE.erl [hunk 9]
```diff
@@ -97,7 +97,7 @@ ftp_220(Config) when is_list(Config) ->
     ok.
 
 ftp_226() ->
-    [{doc, "Especially check that respons can be devided after second status code"
+    [{doc, "Especially check that response can be divided after second status code"
```
Comment (HansN) [inline lib/ftp/test/ftp_format_SUITE.erl:100]:
```suggestion
    [{doc, "Especially check that a response can be divided after second status code"
```

File: lib/ftp/test/ftp_format_SUITE.erl [hunk 10]
```diff
@@ -118,7 +118,7 @@ ftp_257(Config) when is_list(Config) ->
     ok.
 
 ftp_331() ->
-    [{doc, "Especially check that respons can be devided after the third status "
+    [{doc, "Especially check that response can be divided after the third status "
```
Comment (HansN) [inline lib/ftp/test/ftp_format_SUITE.erl:121]:
```suggestion
    [{doc, "Especially check that a response can be divided after the third status "
```

File: lib/ftp/test/ftp_format_SUITE.erl [hunk 11]
```diff
@@ -231,7 +231,7 @@ ftp_multiple_lines(Config) when is_list(Config) ->
 
 ftp_other_status_codes() ->
     [{doc, "Check that other valid status codes, than the ones above, are handled"
-     "by ftp_response:interpret/1. Note there are som ftp status codes" 
+     "by ftp_response:interpret/1. Note there are some ftp status codes" 
      "that will not be received with the current ftp instruction support," 
      "they are not included here."}].
```
Comment (HansN) [inline lib/ftp/test/ftp_format_SUITE.erl:236]:
```suggestion
     "by ftp_response:interpret/1. Note that there are some ftp status codes" 
     "that will not be received with the current ftp instruction support." 
     "They are not included here."}].
```

**Review by HansN — CHANGES_REQUESTED** [review]
I added some grammar comments.

**Comment by kianmeng** [comment]
@HansN You want me to squash the commits and rebased to `maint`?

**Review by HansN — APPROVED** [review]
Looks good.  Please squash to one commit, but keep it on master.

**Comment by kianmeng** [comment]
@HansN Done. Please check.

**Comment by HansN** [comment]
Thanks for the PR.

**Comment by kianmeng** [comment]
:tada: :tada: :tada: :tada: :tada:

**Pull request merged**